app/uk/gov/hmrc/fhregistration/controllers/FhddsApplicationController.scala

```diff
@@ -134,20 +135,23 @@ class FhddsApplicationController @Inject()(
       val event = auditService.buildSubmissionWithdrawalAuditEvent(
         request, fhddsRegistrationNumber)
       auditSubmission(fhddsRegistrationNumber, event)
-      sendEmail(request.emailAddress)
+      sendEmail(
+        request.emailAddress,
+        isWithdrawal = true,
+        emailParameters = Map("withdrawalDate" → new SimpleDateFormat("dd MMMM yyyy").format(processingDate)))
```

Comment:
For date-time processing always prefer Java 8 Date Time API. Whilst not an issue in this case, the Java 8 Date Time offers significant performance improvements over the the older APIs.